chat-widget/src/hooks/useForceEndChat.ts

```diff
@@ -0,0 +1,21 @@
+import { LogLevel, TelemetryEvent } from "../common/telemetry/TelemetryConstants";
+
+import { TelemetryHelper } from "../common/telemetry/TelemetryHelper";
+import useChatSDKStore from "./useChatSDKStore";
+
+const useForceEndChat = () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const chatSDK: any = useChatSDKStore();
+    const forceEndChat = () => {
+        TelemetryHelper.logLoadingEvent(LogLevel.ERROR, {
+            Event: TelemetryEvent.WidgetLoadFailed,
+            ExceptionDetails: {
+                Exception: "SessionInit was successful, but widget load failed."
+            }
+        });
+        chatSDK?.endChat();
```

Comment:
it seems like there is a chance where endChat is not executed , in the case of chatSDK being null... should we log that?